WordPress/Classes/Services/CommentServiceRemoteFactory.swift

```diff
@@ -15,6 +15,10 @@ import WordPressKit
             return CommentServiceRemoteREST(wordPressComRestApi: api, siteID: dotComID)
         }
 
+        if let site = try? WordPressSite(blog: blog) {
```

Comment:
Given [this](https://github.com/wordpress-mobile/WordPress-iOS/pull/24555/files), do we want this to be wired into the app yet? Or should this live alongside until we fix that issue?

Reply:
Sorry, I don't quite understand this comment. The "this" link points to this PR.